skiko/build.gradle.kts

```diff
@@ -582,6 +591,8 @@ library {
     targetMachines.addAll(listOf(machines.macOS.x86_64, machines.linux.x86_64, machines.windows.x86_64))
     baseName.set("skiko-$target")
 
+    putLibraryVersion()
```

Comment:
It makes incorrect Gradle dependency graph and just doesn't belong here. I can make version support in separate PR.